### PR TITLE
Do not prompt to install chromatic script during E2E builds

### PR DESCRIPTION
**Pull request by tevanoff**
The CLI will prompt to install a chromatic script if one does not already exist in `package.json`. This will be the wrong command for E2E builds and may be missing necessary env vars. This disables that for E2E builds for the time being.

<details>
  <summary>📦 Published PR as canary version: <code>11.0.4--canary.941.8178063274.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install chromatic@11.0.4--canary.941.8178063274.0
  # or 
  yarn add chromatic@11.0.4--canary.941.8178063274.0
  ```
</details>

### CODE REVIEW

File: node-src/lib/e2e.ts [hunk 1]
```diff
@@ -59,3 +59,7 @@ export async function getE2eBuildCommand(
     throw err;
   }
 }
+
+export function isE2EBuild(options: Options) {
+  return options.playwright || options.cypress;
+}
```
Comment (skitterm) [inline node-src/lib/e2e.ts:65]: Nice. We could use this for the "flag the project as e2e" thing in the future (if we ever feel the need to handle projects switching from storybook-e2e or vice versa).

File: node-src/index.test.ts [hunk 2]
```diff
@@ -711,6 +713,14 @@ it('prompts you to add a script to your package.json', async () => {
...
 });
 
+it('does not propmpt you to add a script to your package.json for E2E builds', async () => {
+  const ctx = getContext(['--project-token=asdf1234', '--playwright']);
+  await runAll(ctx);
+  const spy = vi.spyOn(checkPackageJson, 'default');
+  expect(spy).not.toHaveBeenCalled();
+  expect(confirm).not.toHaveBeenCalled();
+});
+
```
Comment (skitterm) [inline node-src/index.test.ts:723]: Nice test.

**Review by skitterm — APPROVED** [review]
Looks good!

**Comment by ghengeveld** [comment]
:rocket: PR was released in [`v11.0.4`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.0.4) :rocket:

**Pull request merged**